CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## v0.108.1
```

Comment:
We should probably add the header about upstream versions included here. 👍

Reply:
ba213490